.github/workflows/build.yml

```diff
@@ -23,9 +23,7 @@ env:
 
 jobs:
   build:
-    runs-on: [self-hosted, X64]
-    container:
-      image: ghcr.io/espressosystems/devops-rust:stable
+    runs-on: ubuntu-latest
```

Comment:
Because we need to run docker for the tests now, or?

Reply:
Yeah, and because we don't really need the self-hosted runner for the tests in this repo.

Reply:
It's surprising the build is slightly faster on the public runner.